### PR TITLE
Notes on SLBrain removal in 10.5.1+

### PR DESCRIPTION
**Pull request by wdm-slc**
See DCP234451 / RN40834

### CODE REVIEW

File: user-guide/Advanced_Functionality/DataMiner_Systems/About_DataMiner_Systems/DataMiner_processes.md [hunk 1]
```diff
@@ -154,7 +154,10 @@ The DataMiner Automation process.
 
 ### SLBrain
 
-The DataMiner Correlation process.
+The DataMiner Legacy Correlation process.
+
+> [!NOTE]
+> Only available on DataMiner versions before 10.5.1.
```
Comment (RobbeDGH) [inline user-guide/Advanced_Functionality/DataMiner_Systems/About_DataMiner_Systems/DataMiner_processes.md:160]: i think docs team will require us to put the FR & MR everywhere

**Comment by RobbeDGH** [comment]
Also still a reference to SLBrain.exe [here]([Watchdog logging | DataMiner Docs](https://docs.dataminer.services/user-guide/Troubleshooting/Logging/Consulting_the_DataMiner_logs/DataMiner_logging/Watchdog_logging.html?q=SLBrain))

**Comment by RobbeDGH** [comment]
do we have to do anything about the brainlink table?

**Comment by wdm-slc** [comment]
brainlink table still remains present on SQL setups / central DB schema for now. It contains the linking between actual alarm events and base alarm events even for the newer Correlation Engine

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**